### DISCUSSION TITLE
With No Way to Decode EINs - Is this Data Truly Transparent?

### DISCUSSION THREAD

**Discussion by numbercruncher01** [CMSgov/price-transparency-guide#660] [Q&A]
👉Transparency data holds thousands of EIN & NPI numbers.
👉Most provider contracts are based on EINs.
 
✅ NPIs = complete and public data source available to identify the IDs.
❌ EINs = NO complete and public data source available to identify them.
 
Question…
👉 Only companies with deep pockets can afford +$100K databases to decode the EIN numbers.
👉 **Is this data truly transparent?**

**Comment by dgolden** [comment]
This is a valid point.

Question ... how comprehensive, for medical related entities, are
these +$100K EIN databases?

**Reply by numbercruncher01** [reply]
I would expect them to be pretty complete.  Data providers have access to claims and provider directories from health plans to create these databases.

**Reply by santaviccap** [reply]
Does anybody know what data vendors provide EIN databases?

**Reply by tylerrollinsphd** [reply]
do you know which companies have this service?

**Comment by numbercruncher01** [comment]
I'm about to publish one for hospitals along with their network and ACO affiliations. If you need one for physicians, I'd suggest contacting Komodo or Turquoise.  But be prepared to spend some $$$.

**Reply by dgolden** [reply]
@numbercruncher01 Are you putting this directory into the public domain? That would be much appreciated. You'll post a message to this group when that occurs?

As an aside, I submitted a Freedom of Information Act (FOIA) request with the IRS, asking for access to the EIN database. It was denied. In the follow-up conversation, it was explained to me that the EIN database, even just the EIN # and name,  is considered sensitive information, and they will not release it.

**Comment by lisuhui666** [comment]
Since NPI is a required field, could you use NPIs to identify providers? I guess the downside is that you won't be able to tell the practices that individual NPIs belong to.

**Reply by tylerrollinsphd** [reply]
that's exactly the problem...if you don't know which practice the negotiated rate is at that is a huge issue

**Comment by tdfow** [comment]
The mandate specifies if there is no HIOS identifier, then EIN is the required identification.  Unless everyone is required to have a HIOS, the EIN is the value for identification to meet the Transparency mandate.

(The below is from hhs.gov site.)
For both the In-Network Rate and Out-of-Network Allowed Amount machine-readable files, plans and issuers should start by identifying both the name and the plan identifier for each coverage option offered. The identifier must be one of the following:
(1) If there is a 10-digit Health Insurance Oversight System (HIOS) identifier, this should be used.
(2) If there is no 10-digit identifier, use the 5-digit HIOS identifier.
(3) If there is no HIOS identifier, use the EIN.

26 CFR 54.9815-2715A3(b)(1)(ii)(A) and (iii)(A); 29 CFR 2590.715-2715A3(b)(1)(ii)(A) and (iii)(A); 45 CFR 147.212(b)(1)(ii)(A) and (iii)(A). See also supporting documentation for the information collection approved under OMB Control Number 0938-1372 (CMS-10715, Transparency in Coverage Appendix), available at: https://www.cms.gov/regulations-and-guidancelegislationpaperworkreductionactof1995pra-listing/cms-10715.

**Reply by bryanbriegel** [reply]
Health insurance issuers will have HIOS IDs, and this is a bit confusing because HHS notes "the name and plan identifier" which is actually referring to the 14-byte HIOS ID ( this is the plan level identifier )  With feedback from the market about the redundancy of data and that contracts are executed at the product/network level - the requirements rolled back to the 10-byte HIOS ID, which is the product level (5 byte is the issuer level) -- We would expect health insurance issuers to have HIOS identifiers for their products, and for group health plans, we would expect to see the EIN of the group health plan sponsor.

**Comment by tylerrollinsphd** [comment]
Everything I've looked at has TIN Type = EIN, value = xx-xxxxx. If we don't
have the name of the practice or business that the negotiated rate is
associated with, in addition to the NPI, then we can't know where the rates
are good for/supposed to be applied to. Unless I'm missing something, which
I don't think I am. For example, without EIN/business information we can't
look at the data and show that doctors in X neighborhood are paid less than
doctors in Y neighborhood for the same procedure.

**Reply by chriscalip** [reply]
Confirmed.

Looking my handy overview file file-model-in-network-rates.txt
[file-model-in-network-rates.txt](https://github.com/CMSgov/price-transparency-guide/files/12113586/file-model-in-network-rates.txt)

root->in_network[N]->negotiated_rates are required.
root->in_network[N]->negotiated_rates[N]->negotiated_prices[N] are required.
root->in_network[N]->negotiated_rates[N]->negotiated_prices[N] each record has no associated npi and|or tin.
